Gemfile

```diff
@@ -1,4 +1,26 @@
 source 'https://rubygems.org'
 
-# Specify your gem's dependencies in simhanada_css.gemspec
+# Specify your gem's dependencies in simhanada.gemspec
 gemspec
+
+# Local gems
+
+group :development, :test do
+  ruby '2.3.0'
+
+  gem 'rails', '5.0.0.rc1'
+  gem 'sqlite3'
+
+  # todo: these are defined in the gemspec, but must be used here.
+  gem 'autoprefixer-rails', '>= 6.3.6'
+  gem 'bourbon', '>= 4.2.7'
+  gem 'sass-rails', '>= 5.0.4'
+  gem 'sprockets-es6', '>= 0.9.0'
+end
+
+group :test do
+  gem 'jasmine-rails'
+  gem 'rspec', '~> 3.5.0.beta3'
+  gem 'rspec-core', '~> 3.5.0.beta3'
+  gem 'rspec-rails', '>= 3.5.0.beta3'
```

Comment:
misschien heb je die rspec dependencies helemaal niet nodig. Ik moet alleen `rails s` uitvoeren in het spec/dummy mapje.